.github/workflows/build.yml

```
@@ -25,11 +25,25 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_PULL_USERNAME }}
           password: ${{ secrets.DOCKER_PULL_PASSWORD }}
-      - name: Build image variant "${{ matrix.variant }}"
-        run: cd ${{ matrix.variant }} && docker buildx build ./ -t ${{ matrix.docker-tag }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          buildkitd-flags: --debug
+
+      - name: Build and push image for "${{ matrix.variant }}"
+        uses: docker/build-push-action@v3
+        with:
+          context: ${{ matrix.variant }}
+          platforms: linux/amd64,linux/arm64
```

Given that the list of archs of the base image is wide you should support as much archs too
See: https://hub.docker.com/_/php/tags?page=1&name=8.0-fpm

And here is the full list: https://github.com/sudo-bot/docker-phpmyadmin-snapshots/blob/28d025bff39dd1ded8d581a9e5c079fa3cb9fdf7/.github/workflows/publish.yml#L37

```suggestion
          platforms: linux/386,linux/amd64,linux/arm/v5,linux/arm/v7,linux/arm64/v8,linux/mips64le,linux/ppc64le,linux/s390x
```